moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -588,15 +590,14 @@ bool JogCalcs::checkIfJointsWithinURDFBounds(trajectory_msgs::JointTrajectory& n
 
     // Halt if we're past a joint margin and joint velocity is moving even farther past
     double joint_angle = 0;
-    for (std::size_t c = 0; c < num_joints_; ++c)
+    for (std::size_t c = 0; c < joint_angle = original_jt_state_.size(); ++c)
```

Comment:
Oops, typo here. If you don't mind, I'll make the change

Reply:
fixed

Reply:
That's what I get for managing the commit separately from my working environment :P